### PR TITLE
Clean up dependencies

### PR DESCRIPTION
**Pull request by dougbu**
- remove and sort `using`s
- remove a couple of `project.json` dependencies reached transitively

Was pretty clean before I started this.

### CODE REVIEW

**Comment by kichalla** [comment]
did you mean to merge to release branch?

**Comment by dougbu** [comment]
> did you mean to merge to release branch?

No. This isn't a product change and should go in to `dev` only.

File: build.cmd [hunk 1]
```diff
@@ -33,6 +33,8 @@ IF %BUILDCMD_DNX_VERSION%=="" (
 	CALL packages\KoreBuild\build\dnvm install %BUILDCMD_DNX_VERSION% -runtime CLR -arch x86 -alias default
 )
 CALL packages\KoreBuild\build\dnvm install default -runtime CoreCLR -arch x86
+CALL packages\KoreBuild\build\dnvm install default -runtime CLR -arch x64
```
Comment (kichalla) [inline build.cmd]: These lines were present earlier but not sure why they were removed in this commit : https://github.com/aspnet/MusicStore/commit/19f8830a5746f46b6bb86215b0d6cafb9813c73c

Comment (dougbu) [inline build.cmd]: Well, these lines are mandatory for testing this repo and 19f8830 was a blanket change that overwrote most `build.cmd` files.

**Comment by kichalla** [comment]
:shipit:

**Comment by dougbu** [comment]
f3fb237

**Comment by Eilon** [comment]
Yeah `dev` is right. BTW I have a follow-up change to some of this as well, will ping you guys when it's out.

**Pull request closed without merging**